test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -271,6 +271,89 @@ function ExecuteRequestWithCustomUserAgent {
     return $result
 }
 
+function GetSelfSignedCert {
```

Comment:
We already have a common module to add certificates here:
https://github.com/PowerShell/PowerShell/blob/master/test/powershell/Modules/Microsoft.PowerShell.Security/certificateCommon.psm1

We shouldn't duplicate the code.